utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/dnsmasq_dhcp_leases.lua

```diff
@@ -0,0 +1,30 @@
+local function scrape()
+    local metric_dhcp = metric("dnsmasq_dhcp_lease_expiry_time_seconds", "gauge")
+    f = io.input("/tmp/dhcp.leases")
```

Comment:
Maybe you should not assume that `option leasefile '/tmp/dhcp.leases'` is set in `/etc/config/dhcp`. Maybe just read the config option?

Reply:
Good point should be fixed now :+1: